media.css

```diff
@@ -0,0 +1,457 @@
+@media (min-width: 768px) and (max-width: 1023px) {
+  .wrapper {
+    width: 768px;
+    }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)